index.html

```diff
@@ -116,7 +116,33 @@ <h2 class="about-title">About me</h2>
 <p class="about-contend">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
 <a href="#">Get My Resume</a>
 <div class="ornament2" ></div>
-
 </section>
+
+
+
+
+<footer class="form">
+<form class="form" action="https://formspree.io/f/xaykaovn" method="post">
+<p>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p>
+<input class="form-name" placeholder="Full name" type="text" name="form_name" id="" required>
```

Comment:
- [ ] please ensure that the nameinput has an attribute of maxlength="30". this will limit long names